cmd/virtctl/OWNERS

```diff
@@ -1,8 +1,9 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 reviewers:
   - oshoval
+  - lyarwood
```

Comment:
Would it better to define a SIG subproject or something in the root alias file to avoid the need to do this?

Reply:
(this was just a nit btw so feel free to unhold, I definitely approve the proposal!)